packages/wallet-connectors/CHANGELOG.md

```diff
@@ -7,20 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-### Added
-
--   Build as ES module to facilitate tree-shaking.
-
-## [0.3.2] - 2023-08-17
-
 ### Changed
 
--   Bump dependency `@concordium/web-sdk` to v6.0.0+. This transitively bumps `@concordium/common-sdk` to v9.0.0.
-
-### Fixed
-
--   `WalletConnect`: Fix schema object format conversion in `signAndSendTransaction` in request payloads.
--   `WalletConnect`: Use standard string identifiers for transaction type in request payload.
+-   Build as ES module to facilitate tree-shaking.
 
 ### Removed
```

Comment:
It's actually this `### Removed` section that was moved from `## [0.3.2] - 2023-08-17` to `## [Unreleased]`. And the section of `Build as ES module to facilitate tree-shaking.` was changed from `Added` to `Changed`.

The diff doesn't render the change very clearly so it's probably easier to just compare the two raw files.

Reply:
Or this is better actually https://github.com/Concordium/concordium-dapp-libraries/pull/54/files?short_path=c550704#diff-c55070439909febbe49b01d96c47a47592c7d99db36fed802991f57a915b59fd